dockerfiles/docker-compose-search.yml

```diff
@@ -23,10 +23,12 @@ services:
       - node.name=search
       - cluster.routing.allocation.disk.threshold_enabled=false
       - cluster.info.update.interval=30m
-      - "ES_JAVA_OPTS=-Xms128m -Xmx128m"
       - ELASTIC_PASSWORD=password
       # Disable HTTPS on development.
       - xpack.security.transport.ssl.enabled=false
+      # Make search work on MacOS
+      - 'ES_JAVA_OPTS=-Xms128m -Xmx128m -XX:UseSVE=0'
+      - 'CLI_JAVA_OPTS=-XX:UseSVE=0'
```

Comment:
Looks this broke search for me, I get that UseSVE isn't a valid option for java.